src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -16,6 +16,19 @@ The basic idea is this:
 The triple backticks start and end code blocks. If this were in a file named `foo.rs`,
 running `rustdoc --test foo.rs` will extract this example, and then run it as a test.
 
+Please note that by default, if no language is set for the block code, `rustdoc`
+assumes it is `Rust` code. So the following:
```

Comment:
This block doesn't seem related to the rest of this PR? Like, it's worth noting, but it's not part of "making `compile_fail` stable".

Reply:
Do you want me to put this change in another commit?